styles.css

```diff
@@ -107,6 +107,149 @@ nav li a img {
     width: 100%;
   }
 
+  .mobile-menu {
+    display: none;
+  }
+
+  .popupbackground {
+    width: 100vw;
+    height: 100vh;
+    display: none;
+    z-index: 10;
+    position: fixed;
+    top: 0;
+    background-color: #396df277;
+    backdrop-filter: blur(5px);
+  }
+
+  #demo-mobile {
+    display: none !important;
+  }
+
+  #demo {
+    width: 80%;
+    height: 713px;
+    position: absolute;
+    top: 1%;
+    z-index: 20;
+    display: none;
+    padding: 15px;
+    background-color: #fff;
+    margin-left: 100px;
+    border: 1px solid #ebecf0;
+    margin-top: 30px;
+    overflow: scroll;
+    transition: width 4s;
+  }
```

Comment:
- [ ] Kindly make sure to remove the horizontal scroll in order to match the Figma design as much as possible.

  - ### YOUR DESIGN
  
     ![image](https://user-images.githubusercontent.com/87197412/217769263-4b40dd2a-58ff-414c-a769-92df1934c8f2.png)

Reply:
Suggested changes made horizontal scrollbar removed. Overflow-x-hidden. Please refer to the screen shot attached below for your kind reference. Also Image appearing properly.
![image](https://user-images.githubusercontent.com/117091923/217865162-4c727040-698f-4c5f-b7e4-7101490f542e.png)